namespaces/live-1.cloud-platform.service.justice.gov.uk/moving-people-safely-dev/05-serviceaccount-circleci.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: circleci
+  namespace: moving-people-safely-dev
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: circleci
+  namespace: moving-people-safely-dev
+subjects:
+- kind: ServiceAccount
+  name: circleci
+  namespace: moving-people-safely-dev
+roleRef:
+  kind: ClusterRole
```

Comment:
Can you create a role here , instead of using cluster role, some thing live below
`https://github.com/ministryofjustice/cloud-platform-environments/pull/1865/files`

Reply:
Is there a reason?  That role you link to has pretty much the same permissions.